### ISSUE TITLE
Include exception stack trace args

### ISSUE THREAD

**Issue by bytestream** [rollbar/rollbar-php#88]
It would be great if exception stack trace args were shown in rollbar.

For example:

``` php
#4 [internal function]: Illuminate\Cache\Repository->put('x', Array, Object(Carbon\Carbon))
#5 /vendor/laravel/framework/src/Illuminate/Cache/CacheManager.php(296): call_user_func_array(Array, Array)
#6 /bootstrap/cache/compiled.php(6199): Illuminate\Cache\CacheManager->__call('put', Array)
#7 /bootstrap/cache/compiled.php(6199): Illuminate\Cache\CacheManager->put('x', Array, Object(Carbon\Carbon))
```

**Comment by Crisfole** [comment]
Hey @bytestream your notifier should be doing this already! Can you write into support@rollbar.com with some links to examples where it's not?

**Comment by dlsteuer** [comment]
It looks like this is not currently available in the notifier, re-opening.

**Comment by ArturMoczulski** [comment]
Can't reproduce this issue. Arguments are available in the stack trace in Rollbar UI. This issue has probably been fixed by refactors in v1.0.0.

**Comment by bytestream** [comment]
I'm using v0.18.2

**Comment by rokob** [comment]
Closing because fixed in 1.0
